src/qoption.h

```diff
@@ -0,0 +1,14 @@
+#ifndef QOPTION_H
+#define QOPTION_H
+
+#include <QCoreApplication>
```

Comment:
Let's put only used includes:

```suggestion
#include <QJsonObject>
```
(It should contain `QString` and `QStringList`)